pkg/analyzer/analyzers/github/finegrained/finegrained.go

```diff
@@ -428,11 +428,6 @@ func getDependabotAlertsPermission(client *gh.Client, repo *gh.Repository, curre
 	// Risk: Extremely Low
 	// GET /repos/{owner}/{repo}/dependabot/alerts
 	_, resp, err := client.Dependabot.ListRepoAlerts(context.Background(), *repo.Owner.Login, *repo.Name, &gh.ListAlertsOptions{})
-	if err != nil {
-		if !strings.Contains(err.Error(), "disabled") {
-			return NoAccess, err
-		}
-	}
```

Comment:
It looks like this `err` is completely unchecked now, which seems wrong.

Reply:
It's returned in the default case